internal/driver/server.go

```diff
@@ -45,13 +50,15 @@ func NewNonBlockingGRPCServer() NonBlockingGRPCServer {
 
 // NonBlocking server
 type nonBlockingGRPCServer struct {
-	wg     sync.WaitGroup
-	server *grpc.Server
+	wg            sync.WaitGroup
+	server        *grpc.Server
+	metricsServer *http.Server
 }
 
 func (s *nonBlockingGRPCServer) Start(endpoint string, ids csi.IdentityServer, cs csi.ControllerServer, ns csi.NodeServer) {
 	s.wg.Add(1)
 	go s.serve(endpoint, ids, cs, ns)
+	go s.startMetricsServer(":8081")
```

Comment:
Can we make this dynamic and get the value from the ENV var? May be also only enable metric server if a flag is set to true

Reply:
+1 for this

Reply:
I've pushed the changes to accept flags to enable the metrics